.github/workflows/quality.yaml

```diff
@@ -32,8 +35,7 @@ jobs:
 
       - name: Debug Check (default features)
         run: |
-          git rev-list origin/main..$GITHUB_SHA | xargs -t -I % sh -c 'git checkout %; cargo check --tests --all --target=${{ matrix.target }}'
-          git checkout $GITHUB_SHA
```

Comment:
Is it okay to remove this checkout? I'm asking because `git rev-list` gives the list in reversed order, meaning the last commit that would be checked out will be the first of the potential PR.

Reply:
Fixed. I missed that it was in reverse which now makes sense when it works by looking at parents.